bundles/org.openhab.binding.volumio/src/main/java/org/openhab/binding/volumio/internal/VolumioHandlerFactory.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.volumio.internal;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingTypeUID;
+import org.openhab.core.thing.binding.BaseThingHandlerFactory;
+import org.openhab.core.thing.binding.ThingHandler;
+import org.openhab.core.thing.binding.ThingHandlerFactory;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link VolumioHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Patrick Sernetz - Initial Contribution
+ * @author Chris Wohlbrecht - Adaption for openHAB 3
+ */
+@NonNullByDefault
+@Component(configurationPid = "binding.volumio", service = ThingHandlerFactory.class)
+public class VolumioHandlerFactory extends BaseThingHandlerFactory {
+
+    private final Logger logger = LoggerFactory.getLogger(VolumioHandlerFactory.class);
+
```

Comment:
`[WARNING] .../VolumioHandlerFactory.java:[39,26] The value of the field org.openhab.binding.volumio.internal.VolumioHandlerFactory.logger is not used`

```suggestion
```

Reply:
solved